include/ozo/concept.h

```diff
@@ -253,7 +253,10 @@ template <typename T, typename = std::void_t<>>
 struct is_raw_data_writable : std::false_type {};
 
 template <typename T>
-struct is_raw_data_writable<T, std::void_t<decltype(std::declval<T&>().data())>> : std::true_type {};
+struct is_raw_data_writable<T, std::void_t<
+    decltype(std::declval<T&>().data()),
+    decltype(std::declval<T&>().size())
+>> : std::integral_constant<bool, std::is_same_v<decltype(std::declval<T&>().data()), char*>> {};
```

Comment:
IMO instead of

```cpp
std::integral_constant<bool, std::is_same_v<decltype(std::declval<T&>().data()), char*>>
```

it is better to use 

```cpp
std::bool_constant<sizeof(decltype(*std::declval<T&>().data())) == 1>
```

Or even better - extend this concept like in this commit - https://github.com/yandex/ozo/pull/76/commits/8a3991863c62a08c6ee436c01512fe47c9b80829